CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ All notable changes to this project will be documented in this file, in reverse
 
 ### Fixed
 
-- Nothing.
+- An invalid date string input caused an exception to be thrown by filter(). Added an option to return unfiltered input in the event that the input cannot be formatted.
```

Comment:
This is a new feature and not a fix because the exception was previously a desired result.

Reply:
As I understand it, filters are not expected to throw exceptions at runtime due to input value. From mwop: "Filters generally should NOT throw exceptions; if they cannot handle a given value, they should return it verbatim." BC is maintained.

Reply:
> @marcguyer This is a new feature because the exception was previously a desired result. Please use the develop branch instead.

The question is whether a filter should throw an exception, not whether an exception was desired by the author.

Reply:
@marcguyer 
Right, they "should not" but the current implementation is different in code and tests. Now you add a feature to allow the _correct_ behaviour with backward compatibility in mind.

Reply:
Let me explain it differently: for a bugfix you do not need an extension of documentation, but your changes need to be documented.

Reply:
Changed to develop.